less/opendev.less

```diff
@@ -12,6 +12,7 @@ img {
 }
 
 table {
```

Comment:
@huyeng, let me try to enabling datatables with responsive extension and test how it works